### PR TITLE
Learn more section under each settings

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
> Signed-off-by: Siddhant Khare <siddhant@gitpod.io>

## Description

This PR Propose changes to Settings Dashboard for having `Learn More` section under required settings to make user learn about the feature or about how that actually works by linking it to our documentation. (Design Suggestions & Placeholder position suggestions are welcomed 🙏🏽 cc: @gtsiolis)

## Related Issue(s)

Fixes #

## How to test
- Open [Preview](https://dashboard-074a33b924.preview.gitpod-dev.com/)
- Go to `/settings` & See following 👀
    - Learn More section on `/preferences`
    - Learn More section on `/integrations`
    - Learn More section on `/keys`
    - Learn More section on `/variables`
    - Learn More section on `/billing`
    - Learn More & Send Feedback section on `/tokens`

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by roboquat** [comment]
@Siddhant-K-code: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/dashboard/src/settings/EnvironmentVariables.tsx
Comment (loujaybee) [inline components/dashboard/src/settings/EnvironmentVariables.tsx:229]: **Thought (un-related to PR):** We need to break this env variables docs page down, not all the environment variables that are described on that page are related to the environment variables set here. Hierarchically the content is also under "project" right now (as per the URL) but really this should be `/user/` for this page.

File: components/dashboard/src/components/BillingAccountSelector.tsx
Comment (loujaybee) [inline components/dashboard/src/components/BillingAccountSelector.tsx:109]: **Thought (no action required):** 

* **Pro:** Placing links here means that we can have a "learn more" for each section on a settings page.
* **Con:** Placing here means that 

**Suggestion (non-blocking):** We could elevate to the header area, and potentially use an information icon, as the link blue is quite eye-catching (and I don't think we need it to be super eye-catching. If a user is stuck, they'll actively seek out documentation or help links anyway). A question for @gtsiolis, really. 

image

Comment (Siddhant-K-code) [reply components/dashboard/src/components/BillingAccountSelector.tsx:109]: > We could elevate to the header area, and potentially use an information icon

Agreed ✅. Yeah, George would be the perfect one to answer this 🧡

**Comment by loujaybee** [comment]
Thanks for picking this up @Siddhant-K-code ! 🙏 

Conceptually, it makes a lot of sense to link the relevant docs for these settings page. I'm not 100% sure about the link placement and style, as it feels to me like the anchor links are quite prominent and invite lots of focus on the page. Potentially using a help icon, or putting the learn more link underneath the content (or in the title of the settings page) could improve things, but I'd defer to @gtsiolis on these topics.

**Comment by loujaybee** [comment]
* **Thought:** If this PR is to [address this feedback](https://github.com/gitpod-io/gitpod/issues/15433#issuecomment-1359662657) from @mbrevoort, I don't think we are addressing the feedback fully, as we should also update some of the copy text to describe the feature. Given that the only reasonable use of the access tokens right now is in tandem with the Public API, I suggest we make the required copy changes in this PR, but also that we (at the very least) hide this page from the left navigation (so the URL can be deep linked) but is not exposed to users, I can provide input on what the copy can be updated to 🙏

**Review by easyCZ — APPROVED** [review]
LGTM

/hold in case you'd like @gtsiolis to have a look

**Comment by gtsiolis** [comment]
Looking at this now! 👀

File: components/dashboard/src/settings/PersonalAccessTokens.tsx
Comment (Siddhant-K-code) [reply components/dashboard/src/settings/PersonalAccessTokens.tsx:166]: +1 for this one. Will update it next commit 🙏🏽

File: components/dashboard/src/components/BillingAccountSelector.tsx
Comment (Siddhant-K-code) [reply components/dashboard/src/components/BillingAccountSelector.tsx:109]: Looks Good ✨ 
image

**Comment by gtsiolis** [comment]
Looking at this now! 👀

File: components/dashboard/src/settings/PersonalAccessTokens.tsx
Comment (gtsiolis) [inline components/dashboard/src/settings/PersonalAccessTokens.tsx:174]: **issue**: I skipped mentioning above in https://github.com/gitpod-io/gitpod/pull/15450#discussion_r1061691330 that the links should use the same font size when they are placed next to each other like this.

```suggestion
                            className="gp-link"
```

| BEFORE | AFTER |
|-|-|
| b | a |

File: components/dashboard/src/components/BillingAccountSelector.tsx
Comment (gtsiolis) [inline components/dashboard/src/components/BillingAccountSelector.tsx:107]: **thought**: Added [a report](https://eu.mixpanel.com/s/1K13ak) (internal) to measure how usage (clicks) for these **help links** will change. Cc @jakobhero for visibility and in case we could track this better from inbound traffic to the docs.

File: components/dashboard/src/settings/EnvironmentVariables.tsx
Comment (gtsiolis) [inline components/dashboard/src/settings/EnvironmentVariables.tsx:229]: **thought**: Let's see how fast we can break these links by changes to docs.

**Review by gtsiolis — APPROVED** [review]
LGTM, @Siddhant-K-code!

Left one blocking comment for the feedback link font size. Once updated, let's remove the hold and merge this.

File: components/dashboard/src/components/BillingAccountSelector.tsx
Comment (Siddhant-K-code) [reply components/dashboard/src/components/BillingAccountSelector.tsx:107]: Unholding this PR for now. We can iterate with better solution in follow-up PRs. 🙏🏽

**Pull request merged**
